docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -209,3 +215,71 @@ your application's `web.xml` file, as shown in the example below:
     ...
 </web-app>
 ----
+
+[[identity_propagation]]
+== Identity Propagation
+
+When securing an application with OpenID Connect, the `elytron-oidc-client` subsystem will automatically
+create a virtual security domain for you. If your application invokes an EJB, additional configuration
+might be required to propagate the security identity from the virtual security domain depending on how
+the EJB is being secured.
+
+=== Securing an EJB using a different security domain
+
+If your application secured with OpenID Connect invokes an EJB within the same deployment (e.g.,
+within the same WAR or EAR) or invokes an EJB in a separate deployment (e.g., across EARs)
+and you'd like to secure the EJB using a different security domain from your servlet,
+additional configuration will be needed to outflow the security identities established by
+the virtual security domain to another security domain.
+
+The `virtual-security-domain` resource allows you to specify that security identities
+established by a virtual security domain should automatically be outflowed to other
+security domains. A `virtual-security-domain` resource has two attributes, as
```

Comment:
There is also `outflow-anonymous` related and available.

Reply:
Updated, thanks!